templates/zerver/organization_signup.html

```diff
@@ -0,0 +1,56 @@
+{% block portico_content %}
+
+<div class="input-box">
+    <div class="input-box">
+        <label class="inline-block">{{ _('Organization type') }}</label>
+    </div>
+    <div id="organization_type_select_section" class="input-group inline-block">
+        <select class="organization_type" name="organization_type" id="organization_type_select">
+            <option value="Corporate">{{_('Corporate')}}</option>
+            <option value="Open-source">{{_('Open Source')}}</option>
+            <option value="Education">{{_('Education')}}</option>
+            <option value="Non-profit">{{_('Non-Profit')}}</option>
+            <option value="Other">{{_('Other')}}</option>
+        </select>
+    </div>
+</div>
+<div id="offer_details">
+    <div class="input-box">
+        <div class="help-box margin-top" id="offer_details_help_box"></div>
+        <div id="open_source_offer">
+            <div class="input-box">
+                <label for="repository_link_label" class="inline-block label-title">{{ _('Repository Link') }}</label>
+                <div class="inline-block relative">
+                    <input id="repository_link_label" class="required" type="text"
+                      name="repository_link" required />
+                </div>
+            </div>
+        </div>
+        <div id="non_profit_offer">
+            <div class="input-box">
+                <label for="organization_nature_label" class="inline-block label-title">{{ _('Briefly describe the nature of your organization') }}</label>
```

Comment:
Both "non-profit" and other should just do "organization description", instead of having separate "organization nature" for non-profit - i don't think such a distinction is relevant

Reply:
This still isn't resolved, we have:
`<label for="organization_nature_label" class="inline-block label-title">{{ _('Briefly describe the nature of your organization') }}</label>`
and below:
`<label for="reasons_label" class="inline-block label-title">{{ _('Briefly explain about your organization') }}</label>`

I don't think there's a reason for these to be distinct at all? Should just be same label and text? I think `Briefly describe the nature of your organization` is the better phrasing of the two

Reply:
Oh, I misunderstood it previously with changing fields in the non-profit category. Made the changes now.